src/Illuminate/Database/Console/Seeds/SeedCommand.php

```diff
@@ -33,17 +34,25 @@ class SeedCommand extends Command
      */
     protected $resolver;
 
+    /**
+     * Timer instance.
+     *
+     * @var \Illuminate\Support\Stopwatch
+     */
+    protected $watch;
+
     /**
      * Create a new database seed command instance.
      *
-     * @param  \Illuminate\Database\ConnectionResolverInterface  $resolver
-     * @return void
+     * @param \Illuminate\Database\ConnectionResolverInterface $resolver
+     * @param \Illuminate\Support\Stopwatch $watch
      */
-    public function __construct(Resolver $resolver)
+    public function __construct(Resolver $resolver, Stopwatch $watch)
```

Comment:
Do we need the dependency here? StopWatch doesn't has any dependencies so we could just resolve it inside the class. It would save you from having to set-up everything in the service provider and add all the `setWatch` methods.